packages/pg/lib/defaults.js

```diff
@@ -59,6 +59,10 @@ module.exports = {
   // false=unlimited
   statement_timeout: false,
 
+  // Terminate any session with an open transaction that has been idle for longer than the specified duration in milliseconds
+  // false=unlimited
+  idle_in_transaction_session_timeout: false
```

Comment:
you need a comma at the end of the line here

Reply:
Doh, thanks! Was having trouble getting the tests running locally so relied on running on travis, thanks for the quick look!

Reply:
yeah no problemo!  The tests are pretty antiquated since a lot were written before things like mocha and jest existed. If you have vscode & docker installed you can install the `remote - containers` extension & it'll boot you into an isolated environment where you can develop & test....hopefully that helps!  Using travis to run tests is also fine with me!